src/stats.cpp

```diff
@@ -0,0 +1,90 @@
+
+#include <pybind11/pybind11.h>
+#include <pybind11/numpy.h>
+#include <cmath>
+
+namespace py = pybind11;
+
+double erfinv(double x);
+double normal(double x);
+double truncnorm(double x, double alpha, double beta);
+
+constexpr double sqrt2 = 1.41421356237309504880168872420969807856967187;
+constexpr double erfinv_a3 = -0.140543331;
+constexpr double erfinv_a2 = 0.914624893;
+constexpr double erfinv_a1 = -1.645349621;
+constexpr double erfinv_a0 = 0.886226899;
+ 
+constexpr double erfinv_b4 = 0.012229801;
+constexpr double erfinv_b3 = -0.329097515;
+constexpr double erfinv_b2 = 1.442710462;
+constexpr double erfinv_b1 = -2.118377725;
+constexpr double erfinv_b0 = 1;
+
+constexpr double erfinv_c3 = 1.641345311;
+constexpr double erfinv_c2 = 3.429567803;
+constexpr double erfinv_c1 = -1.62490649;
+constexpr double erfinv_c0 = -1.970840454;
+
+constexpr double erfinv_d2 = 1.637067800;
+constexpr double erfinv_d1 = 3.543889200;
+constexpr double erfinv_d0 = 1;
+
+
+double normal(double x)
+{
+    return sqrt2*erfinv(2*x - 1);
+}
+
+
+double truncnorm(double x, double alpha = -INFINITY, double beta = INFINITY)
```

Comment:
Can you add a (very brief) description of the parameters?

Reply:
See line https://github.com/vetlewi/ompy/blob/0beac1ee1796ebe6071fa327bc597e63919b6dc2/src/stats.cpp#L43-L56